### PR TITLE
Button bug fixes sprint 3

### PR DESCRIPTION
**Pull request by dkwhite200**

### CODE REVIEW

**Review by eristow — APPROVED** [review]
Everything looks good to me

File: modules/clientmanagements/client/views/create-clientmanagement.client.view.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
         <br>
         <div class="form-group">
           <div class="form-group pull-right">
-          <button type="submit" class="btn btn-default" href="/client/views/view-clientmanagement.client.view.html">{{vm.clientmanagement._id ? 'Update' : 'Create'}}</button>
```
Comment (wcampbell09) [inline modules/clientmanagements/client/views/create-clientmanagement.client.view.html:56]: ???

File: modules/clientmanagements/client/views/edit-clientmanagement.client.view.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                <br>
+                <label class="control-label" for="name">Rank:</label>
+                <input name="rank" type="text" ng-model="vm.clientmanagement.rank" id="rank" class="form-control" placeholder="Rank" required>
+                <div ng-messages="vm.form.clientmanagementForm.name.$error" role="alert">
+                    <p class="help-block error-text" ng-message="required">The name entry must be entered. Thank you.</p>
+                </div>
+                <br>
+                <div class="form-group">
+                    <div class="form-group pull-right">
+                        <input type="submit" value="Update Client" class="btn btn-primary" />
```
Comment (wcampbell09) [inline modules/clientmanagements/client/views/edit-clientmanagement.client.view.html:59]: Ah I see...

**Review by wcampbell09 — APPROVED** [review]
looks good to me

File: modules/clientmanagements/client/views/create-clientmanagement.client.view.html (hunk 1 above)
Comment (dkwhite200) [reply modules/clientmanagements/client/views/create-clientmanagement.client.view.html:56]: Since it is a form, you can just have a submit button.

**Pull request merged**